rosdep/python.yaml

```diff
@@ -135,6 +135,30 @@ pika:
     pip:
       packages: [pika]
   ubuntu: [python-pika]
+pyapi-gitlab:
```

Comment:
waiting for feedback on the necessity to add these keys.
If we need to add them, all the commencts below apply here as well